core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.service.ts

```diff
@@ -43,42 +43,44 @@ export class TopicsAndSkillsDashboardPageService {
       topicsArray: Array<ITopicSummaryBackendDict>,
       filterObject: TopicsAndSkillsDashboardFilter):
       Array<ITopicSummaryBackendDict> {
-    const {sort, keyword, category, status} = filterObject;
+    // const {sort, category, status} = filterObject;
```

Comment:
Again, before asking for a review, **please self-review your PR**. Commented-out code should not be included in PRs.

Reply:
Done.